test/gold/tools/ethNames/api_tests/ethNames_edit_add.txt

```diff
@@ -47,30 +47,30 @@ Env: TB_NAME_SOURCE   = SOURCE
 Env: TB_NAME_SYMBOL   = SYMBOL
 Env: TB_NAME_DECIMALS = 11
 Env: TB_NAME_DESCR    = DESCRIPTION
+CURL: -X POST
```

Comment:
Wonderful!

Reply:
For the record: each test using custom curl options require `[test_name]_curl.txt` file, which we can feed to curl when debugging: `curl --config some_test_curl.txt http://localhost:8080`